locales/en/cos-ui.json

```diff
@@ -82,11 +83,13 @@
   "createConnector": "Create Connector",
   "createConnectorsInstance": "Create Connectors instance",
   "created": "Created",
+  "createOSDTrail": "Create a namespace on an OpenShift Dedicated trial cluster",
```

Comment:
I'm just now realizing that I think the contents we meant to use were modified in the design document and I didn't catch that change. 

In a previous chat, we were hoping to use the acronyms in the empty state cards to avoid having these be really long. I'll work with Katie and Melissa about getting the final content updates in place, but I would suggest we go ahead and change this text to the following:

> Create a namespace on an OSD trial cluster

Reply:
ok will update it to "Create a namespace on an OSD trial cluster" 👍